### PR TITLE
chore: remove unused plugins, fix go.mod

### PR DESCRIPTION
**Pull request by bengerman13**
Worth noting that this fails two of its unit tests *but* it fails the same two before my changes so 🤷🏻‍♂️ 

## changes proposed
- remove kubernetes plugin, since we don't use it (you can see [here](https://github.com/cloud-gov/cg-deploy-stratos/blob/main/stratos.yaml#L3) that we're not using it, and [in the docs](https://stratos.app/docs/extensions/disable-packages#remove-by-deletion) it says you can delete unused plugins)
- remove desktop plugin, since we don't use it (you can see [here](https://github.com/cloud-gov/cg-deploy-stratos/blob/main/stratos.yaml#L4) that we're not using it, and [in the docs](https://stratos.app/docs/extensions/disable-packages#remove-by-deletion) it says you can delete unused plugins)
- revert to cloudfoundry-community version of go-cfenv since the app won't compile otherwise
- fix type casting on an error message
- bump dependency versions

## security considerations
Dropping these plugins will cut some noise from dependabot alerts

### CODE REVIEW

File: src/jetstream/go.mod [hunk 1]
```diff
@@ -117,59 +111,46 @@ replace (
 	k8s.io/sample-apiserver => k8s.io/kubernetes/staging/src/k8s.io/sample-apiserver v0.0.0-20191001043732-d647ddbd755f
 )
 
+require github.com/cloudfoundry-community/go-cfenv v1.18.0
```
Comment (markdboyd) [inline src/jetstream/go.mod:114]: I don't think we can do this. Part of the fallout from the security incident with the `cloudfoundry-community` org was that we had to remove all references to that org in our code. What errors are we getting from trying to use our fork?

Comment (bengerman13) [reply src/jetstream/go.mod:114]: this is causing a type collision [here](https://github.com/cloud-gov/stratos/blob/main/src/jetstream/main.go#L111-L113)

Comment (bengerman13) [reply src/jetstream/go.mod:114]: we can fix this with cloud-gov/cf-common#1

Comment (bengerman13) [reply src/jetstream/go.mod]: updated - we no longer rely on cf community code

**Review by markdboyd — APPROVED** [review]
Great work @bengerman13 !

**Pull request merged**